iphone/Classes/MediaModule.m

```diff
@@ -1863,7 +1863,8 @@ - (void)imagePickerController:(UIImagePickerController *)picker_ didFinishPickin
             }
             
             if (resultImage == nil) {
-                resultImage = (editedImage != nil) ? editedImage : originalImage;
+                UIImage *tempImage = (editedImage != nil) ? editedImage : originalImage;
```

Comment:
The `tempImage` can leak, please release it after using or simplify it by using:
```objc
resultImage = [TiUtils adjustRotation:editedImage || originalImage];
```
Not 100 % if that works but it should.